### PR TITLE
Implement ES resolvers

### PR DESCRIPTION
**Pull request by bboure**
This is WIP. I still have to add some tests.

### CODE REVIEW

File: packages/appsync-emulator-serverless/elasticsearchSource.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const elasticsearchSource = async (
+  endpoint,
+  { path = '', operation = 'POST', params = {} },
+) => {
+  const esResult = await httpSource(endpoint, {
+    // ES accepts GET/HEAD operations with a body,
+    // but fetch forbids it.
+    method:
+      ['HEAD', 'GET'].indexOf(operation) !== -1 &&
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/elasticsearchSource.js]: nitpick: you can use includes: `['HEAD', 'GET'].includes(operation)`

**Comment by bboure** [comment]
@cbaron I added tests.
Also tested locally with a local ES instance.
All looks good

File: packages/appsync-emulator-serverless/__test__/elasticSearchSource.test.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        headers: { 'Content-Type': 'application/json' },
+        query: undefined,
+      },
+      resourcePath: '/test/_search',
+    });
+  });
+
+  it('HTTP GET gets transformed to POST', async () => {
+    await elasticsearchSource('http://localhost:9200', {
+      operation: 'POST',
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/__test__/elasticSearchSource.test.js]: should this be `GET` ?

File: packages/appsync-emulator-serverless/elasticsearchSource.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+const elasticsearchSource = async (
+  endpoint,
+  { path = '', operation = 'POST', params = {} },
+) => {
+  const esResult = await httpSource(endpoint, {
+    // ES accepts GET/HEAD operations with a body,
+    // but fetch forbids it.
+    method:
+      ['HEAD', 'GET'].includes(operation) !== -1 &&
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/elasticsearchSource.js]: I think we want:

`['HEAD', 'GET'].includes(operation) &&`

?

Comment (bboure) [reply packages/appsync-emulator-serverless/elasticsearchSource.js]: ups, yes

**Comment by bboure** [comment]
@cbaron I fixes issues and added test to cover them.
I'll give you a chance to re-review. If all ok, I'll bump later today.

**Comment by cbaron** [comment]
LGTM!

**Comment by bboure** [comment]
@cbaron what are your criteria for version bump?
Should we bump MINOR version after this change?

**Comment by cbaron** [comment]
@bboure -- yeah this feels like a MINOR to me 👍

**Comment by cbaron** [comment]
I don't have any hard rules for version bumping.

**Pull request merged**